db/migrations/0002_alter_player_guild.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2023-12-14 22:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild'),
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app in a pull request with all changes made to the models within one app. It helps to not overload the `migrations` folder, so include one migration to this PR.